contentctl/output/conf_output.py

```diff
@@ -152,6 +152,10 @@ def writeObjects(self, objects: list, type: SecurityContentType = None) -> set[p
                                     'macros.j2',
                                     self.config, objects))
         
+        elif type == SecurityContentType.dashboards:
+            written_files.update(ConfWriter.writeDashboardFiles(self.config, objects))
```

Comment:
Dashboards are only written to the app.
They do not have an equivalent JSON output since nothing (SSE) can do anything with them

Reply:
LGTM